.github/workflows/release-windows.yml

```
@@ -22,11 +22,13 @@ jobs:
 
         strategy:
             matrix:
-                arch: [x64]
+                arch: [x64, arm64]
                 flavor: [debug, release]
                 include:
                     - arch: x64
                       rust_target_prefix: x86_64
+                    - arch: arm64
+                      rust_target_prefix: aarch64
```

```suggestion
                    - arch: x64
                      runs_on: windows-latest
                      rust_target_prefix: x86_64
                    - arch: arm64
                      runs_on: windows-11-arm
                      rust_target_prefix: aarch64
```

Maybe this? Then use `${{ matrix.runs_on }}` above?

---

That's better, thanks!